django_icons/__init__.py

```diff
@@ -38,3 +38,30 @@ def icon(name, *args, **kwargs):
     renderer_class = get_icon_renderer(icon_kwargs.get("renderer", None))
     renderer = renderer_class(**icon_kwargs)
     return renderer.render()
+
+
+def icon_copyright(renderer=None, *args, **kwargs):
```

Comment:
In mapping context, what you call copyright is usally called 'attribution' I think that's a more appropriate term here, although I'm happy to be corrected.

Reply:
I think you're right, copyright is maybe not the most suitable term. I was thinking to 'citation', which is commonly used when referencing.

Reply:
I was thinking of 'source', but in the end I find 'attribution' more accurate